### PR TITLE
Cu 341503c token generation blessing obasohab

### PR DESCRIPTION
**Pull request by OsasTheCoder**
**What does this pull request do?**

      Create Generate and refresh Token function

**Description of tasks completed**
- installs dependencies
- setup database 
- created the function

**How to test?**
- clone the repo
- install the dependencies by running `npm i`
- run `npm run dev` to start the server

### CODE REVIEW

**Comment by Endowmissy** [comment]
Task linked: [CU-341503c Token Generation](https://app.clickup.com/t/341503c)

**Comment by Endowmissy** [comment]
Task linked: [CU-3wekbxf Logger Implementation](https://app.clickup.com/t/3wekbxf)

File: src/utils/jwt.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+/**
+ *
+ */
+export default class jwtHelper {
+  /**
+   * @param {object} payload - The details to be signed
+   * @param {string} secret - The JWT secret key
+   * @returns {string} The JWT signed token
+   */
+  static async generateToken(payload, secret = secretKey) {
```
Comment (kelue) [inline src/utils/jwt.js:15]: payload is not defined

Comment (OsasTheCoder) [reply src/utils/jwt.js:15]: payload is the data you pass into the function
the data will be the user data

**Pull request closed without merging**